src/main/java/io/vertx/mqtt/impl/MqttEndpointImpl.java

```diff
@@ -295,14 +306,19 @@ public MqttEndpointImpl exceptionHandler(Handler<Throwable> handler) {
     }
   }
 
-  private MqttEndpointImpl connack(MqttConnectReturnCode returnCode, boolean sessionPresent) {
+  static io.netty.handler.codec.mqtt.MqttMessage createConnack(MqttConnectReturnCode returnCode, boolean sessionPresent) {
```

Comment:
This method isn't needed anymore if we are going to create an `MqttEndpointImpl` instance so we can just call the `connack` method as before.